PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx

```diff
@@ -2621,7 +2621,9 @@ void AliCaloPhotonCuts::FillHistogramsExtendedQA(AliVEvent *event, Int_t isMC)
           fHistCellEnergyvsCellID->Fill(cellAmplitude,cellNumber);
       if(fHistCellEnergyvsCellID && (cellAmplitude > 0.01) && fClusterType == 2 )
           fHistCellEnergyvsCellID->Fill(cellAmplitude,cellNumber);
-      if(fHistCellTimevsCellID && (cellAmplitude > 0.2)) fHistCellTimevsCellID->Fill(cellTime,cellNumber);
+      if(fHistCellTimevsCellID && (cellAmplitude > 0.01) && fClusterType == 2 )
+          fHistCellTimevsCellID->Fill(cellTime,cellNumber);
+      else if(fHistCellTimevsCellID && (cellAmplitude > 0.2) && fClusterType != 2) fHistCellTimevsCellID->Fill(cellTime,cellNumber);
```

Comment:
the && fClusterType != 2 would not have been necessary here

Reply:
yes, i deleted it